### PR TITLE
Added DELETE Support

### PR DESCRIPTION
**Pull request by naoyukik**
Added DELETE and test.

Thank you!

### CODE REVIEW

File: test/TwitterAPIExchangeTest.php [hunk 1]
```diff
@@ -322,4 +322,23 @@ public function testPut()
...
+        $account_data = $this->exchange->request($url, $method, $params);
+        $id = json_decode($account_data)->data->id;
+
+        // Request delete
+        $url = $url.'/'.$id;
+        $method = 'DELETE';
+        $account_data = $this->exchange->request($url, $method, $params);
+        $deleted = json_decode($account_data)->data->deleted;
+
+        $this->assertTrue($deleted);
```
Comment (J7mbo) [inline test/TwitterAPIExchangeTest.php]: This isn't a good test - you need to assert that we actually get valid data back through and not unauthorised.

**Comment by J7mbo** [comment]
Also, your test doesn't actually pass yet :-)

**Comment by J7mbo** [comment]
Great, thanks! If you could squash your commits into just 'Added DELETE support', I'll merge to develop.

**Comment by naoyukik** [comment]
I finished squash.
Thank you for your support!

**Comment by J7mbo** [comment]
Great, merging in now, cheers!

**Pull request merged**